app/assets/javascripts/discourse/app/components/composer-container.gjs

```diff
@@ -32,6 +33,11 @@ export default class ComposerContainer extends Component {
   @service composer;
   @service site;
 
+  @action
+  updateTags(newTags) {
+    this.composer.model.set("tags", [...newTags]);
```

Comment:
what is this achieving compared to previous code?

Reply:
Since the previous code mutatted the array the UI wasn't updated

Reply:
I don't see anything broken if I remove it